cciskel-duffy

```diff
@@ -124,7 +124,7 @@ def write_inventory(args, allocated_class, topo_data=None):
             n_allocated = 0
             for machine in allocated_class:
                 if machine['job'] != '':
-                    continue
+                    pass
```

Comment:
I'm confused by this...doesn't it break the logic for avoiding grabbing machines already taken by a job?